paths/graphql/05-another-query.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Use a Query to Find Specific Information
+permalink: /graphql/another-query
+next-page: /graphql/mutation
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  Let's build another query. We'll ask GitHub for the ID of an issue so that we can post to that issue using the API in a later step.
+  
+  ![img](../images/gifs/graphql/get-issue-id.gif)
+
+  1. Build a query to find [this issue](https://github.com/githubschool/graph-ql/issues/1)'s unique identifier, or `id`.
```

Comment:
@github/services-training I made a game time decision in cbfcc30 to change the activity to be more sustainable. Now, it doesn't specify any issue, but if they're looking for one to contribute to, they can open one in the graph-QL repository. We can change this of course, but it seems closer and we needed some direction other than everyone commenting on the same issue.